ci: add CMAKE_EXTRA_PARAMS to LuaJIT workflow

igormunkin opened the pull request:
It's quite inconvenient to extend LuaJIT integration workflow now: one need to patch GitHub workflow file in Tarantool repository and then setup integration testing in LuaJIT repository.

This patch introduces a new workflow parameter that can replace several existing paramters (e.g. buildtype and GC64) and allow to easy extend integration CI in LuaJIT (with rare little touches in Tarantool).

NO_DOC=ci
NO_TEST=ci
NO_CHANGELOG=ci

P.S. Test results of the changes made can be found here: tarantool/luajit@c807be7.
P.P.S Examples of `CMAKE_EXTRA_PARAMS` usage can be found here: tarantool/luajit@227d1b0 and tarantool/luajit@65b7d7c.

Buristan reviewed (approved):
Hi, Igor!
Thanks for the patch!
LGTM, with the single nit regarding the commit message.
>  one need to patch GitHub workflow file in Tarantool repository and then

Typo: s/need/needs/

Side note: I see no usage of `FIXUP:` tag in the Tarantool code. Do you mean `FIXME:`  or `TODO:`?

igormunkin commented:
Sergey,

Thanks for your review!

> Hi, Igor! Thanks for the patch! LGTM, with the single nit regarding the commit message.

Added your tag:
| Reviewed-by: Sergey Kaplun <skaplun@tarantool.org>

> 
> > one need to patch GitHub workflow file in Tarantool repository and then
> 
> Typo: s/need/needs/

Fixed.

> 
> Side note: I see no usage of `FIXUP:` tag in the Tarantool code. Do you mean `FIXME:` or `TODO:`?

Definitely `FIXME:`, thanks.

ylobankov reviewed (approved):
LGTM

igormunkin commented:
Yaroslav,

Thanks for your review!

> LGTM

Added your tag:
| Reviewed-by: Yaroslav Lobankov <y.lobankov@tarantool.org>
